graylog2-web-interface/src/views/components/sidebar/FieldList.jsx

```diff
@@ -72,10 +72,13 @@ const FieldList = createReactClass({
   _updateHeight() {
     const fieldsContainer = this.fieldList;
 
-    const maxHeight = this.props.maximumHeight
-      - fieldsContainer.getBoundingClientRect().top;
+    const { maximumHeight } = this.props;
 
-    this.setState({ maxFieldsHeight: Math.max(isNaN(maxHeight) ? 0 : maxHeight, this.MINIMUM_FIELDS_HEIGHT) });
+    if (fieldsContainer && fieldsContainer.getBoundingClientRect) {
+      const maxHeight = maximumHeight - fieldsContainer.getBoundingClientRect().top;
+
+      this.setState({ maxFieldsHeight: Math.max(isNaN(maxHeight) ? 0 : maxHeight, this.MINIMUM_FIELDS_HEIGHT) });
```

Comment:
should we use `Number.isNan` here?

Reply:
No. While the code is not good, type coercion of `isNaN` (which `Number.isNaN` does not perform) is actually desired, as it would require checks for `undefined` et al otherwise. I think we should replace it with `lodash.isFinite` instead though.